src/bitcoin-chainstate.cpp

```diff
@@ -49,7 +52,11 @@ int main(int argc, char* argv[])
     SelectParams(CBaseChainParams::MAIN);
     const CChainParams& chainparams = Params();
 
-    init::SetGlobals(); // ECC_Start, etc.
+    kernel::Context kernel_context{};
+    // We can't use a goto here, but we can use an assert since none of the
```

Comment:
unrelated: Probably a good time to get rid of `goto` instead of adding comments on why `goto` can't be used here. :sweat_smile: